aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -1047,6 +1047,17 @@ pub enum EntryFunctionCall {
         contract_address: AccountAddress,
     },
 
+    VestingWithoutStakingCreateVestingContractWithAmounts {
+        shareholders: Vec<AccountAddress>,
+        amounts: Vec<u64>,
+        schedule_numerator: Vec<u64>,
+        schedule_denominator: Vec<u64>,
```

Comment:
We need only one denominator, not a vector.

Reply:
addressed in new commit.